### ISSUE TITLE
Cash

### ISSUE THREAD

**Issue by jamesgeddes** [GeekZoneHQ/web#114]
Allow members to pay by direct bank deposit of cash into the [Geek.Zone/Bank](http://Geek.Zone/Bank) account

**Comment by CarwynNelson** [comment]
Geek.Zone would need a bank account (Like Monzo, or I guess Starling) that has an API for us to be able to automate this.

Even if that is just an account that receives membership payments (via something like a standing order) and then periodically the money gets transferred to the main bank account.

**Comment by jamesgeddes** [comment]
For accessibility, we would need a bank that has highstreet branches, so we would probably not be able to use Starling. Thankfully https://www.openbanking.org.uk is a thing but certainly not as easy as Starling.

**Comment by CarwynNelson** [comment]
The tricky thing with open banking is getting access to the API for your own account. For some reason most banks will allow a third party access to your account via an API, but will not let you access your own account with that same API.

It's a very strange situation.

In terms of high street branches, can't folks pay into any bank account from their high street bank?

Let's say I bank with Barclays and Geek.Zone has a Monzo account, surely I can walk into a Barclays branch and ask them to pay money into the Geek.Zone Monzo account?

**Comment by jamesgeddes** [comment]
I have just called Barclays customer services, who advised that,
* Barclays branches can only deposit funds into Barclays accounts unless a (complicated) agreement is put in place between Barclays and NatWest for our account.
* A paying-in slip or a bank card is now required for _all_ deposits due to the [The Money Laundering, Terrorist Financing and Transfer of Funds (Information on the Payer) Regulations 2017](https://www.legislation.gov.uk/uksi/2017/692/contents/made) (I think paragraph [28(2)(a)](https://www.legislation.gov.uk/uksi/2017/692/regulation/28/made#regulation-28-2-a:~:text=identify%20the%20customer%20unless%20the%20identity,been%20verified%20by%2C%20the%20relevant%20person) is particularly relevant (IANAL))

The last time I tried to pay cash into someone else's bank account was long before 2017, so I was not aware that this had legislation had been implemented until they informed me. #themoreyouknow ay?

Paying-in slips are unique so we cannot publish a generic one that members can print off and fill in. If we wanted to allow people to pay into the GZ bank account, we would have to give everyone a paying-in book which is clearly not feasible.

In short, it is highly unlikely that we will be able to offer full self-service support to unbanked individuals, so we will encourage cashless transactions. The only solution I can think of would be calendly (or similar) integration to book IRL time with a Trustee, likely during an IRL GZ event like Pint Night.

Marked as P4 as this is,

- a technically and legally "hard problem", so is one that we are unlikely to be able to resolve.
- an edge case

**Comment by jamesgeddes** [comment]
[PayPoint](https://serviceprovider.paypoint.com/) might be the solution here. I have made contact with them, so once I know some details I will upate.

**Comment by jamesgeddes** [comment]
PayPoint only works with larger organisations at this time, as they currently require bespoke development for every implementation. They have a £25k setup fee and a monthly service charge. This is clearly not feasible for GZ at this time. I suggested to PayPoint that if they were able to pull together a generic product that does not require bespoke development every time, this would help many thousands of smaller organisations such as us who need to support their unbanked stakeholders, which would, in turn, create more revenue for PayPoint. PayPoint advised that this is not something they can offer at this time but that they would investigate the feasibility.

Additionally, to provide an e-wallet to members, regardless of the payment channel, we would require an [e-money licence](https://www.fca.org.uk/firms/apply-emi-payment-institution), or would need to use an umbrella e-money licence provider such as [PayrNet](https://www.railsbank.com/payrnet). Either way, this will take a significant time investment.

As far as I am aware, the only other solution that would support unbanked members would be for the member to obtain a pre-paid cash card. Unfortunately, this puts the onus back onto the individual, rather than being something that GZ can help with.

If anyone can think of any other solutions to this problem, I am all ears. It is important that we solve this as we cannot have cash on the premises at [Geek.Zone/01](http://Geek.Zone/01).
